libs/application/templates/financial-aid/src/forms/Apply.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  buildForm,
+  buildDescriptionField,
+  Form,
+  FormModes,
+} from '@island.is/application/core'
+
+export const Apply: Form = buildForm({
```

Comment:
Should we name this FinancialAidApplication or just Application instead of Apply? Just a thought